fuse_tutorials/src/range_sensor_simulator.cpp

```diff
@@ -117,26 +117,26 @@ std::vector<Beacon> createNoisyBeacons(const std::vector<Beacon>& beacons)
 /**
  * @brief Convert the set of beacons into a pointcloud for visualization purposes
  */
-sensor_msgs::PointCloud2::ConstPtr beaconsToPointcloud(
+sensor_msgs::msg::PointCloud2::ConstSharedPtr beaconsToPointcloud(
```

Comment:
These methods are constructing ROS messages to be published. In ROS1 I always publish messages as shared ptrs. But it seems you have been publishing and subscribing with bare objects in ROS2. If that is the case, you can drop the shared pointer complexity here entirely.